### ISSUE TITLE
Distribution mismatch between validation and test splits of How2QA

### ISSUE THREAD

**Issue by phseo** [ych133/How2R-and-How2QA#3]
Thanks for your dataset.

I am currently working on your dataset but all my models clearly show different tendencies on validation and test sets.
As the models start to converge on the training set, the accuracy clearly increases in the validation set. However, in the test set, the accuracy seems to decrease from a very early stage. It seems like there is a clear discrepancy between the validation and test sets. Could you please confirm if the proposed HERO model shows similar behaviours on these validation and test subsets? Also, would it be possible to get the validation performances of the models? Are the validation scores the same as in the v1 paper?

You can find the accuracy curves for training / validation / test sets [here](https://drive.google.com/file/d/1YPkalOEoo718sQW0MV1emDAQT6_D_8Kl/view?usp=sharing).

Thank you for your help in advance.

**Comment by ych133** [comment]
> Thanks for your dataset.
> 
> I am currently working on your dataset but all my models clearly show different tendencies on validation and test sets.
> As the models start to converge on the training set, the accuracy clearly increases in the validation set. However, in the test set, the accuracy seems to decrease from a very early stage. It seems like there is a clear discrepancy between the validation and test sets. Could you please confirm if the proposed HERO model shows similar behaviours on these validation and test subsets? Also, would it be possible to get the validation performances of the models? Are the validation scores the same as in the v1 paper?
> 
> You can find the accuracy curves for training / validation / test sets [here](https://drive.google.com/file/d/1YPkalOEoo718sQW0MV1emDAQT6_D_8Kl/view?usp=sharing).
> 
> Thank you for your help in advance.

We separated the batches of train, val and test sets when collecting the data from AMT.  Therefore, there are some distribution mismatches among them. We also observed this from HERO. We will release the binary prediction of test/val separately.

**Comment by ych133** [comment]
> Thanks for your dataset.
> 
> I am currently working on your dataset but all my models clearly show different tendencies on validation and test sets.
> As the models start to converge on the training set, the accuracy clearly increases in the validation set. However, in the test set, the accuracy seems to decrease from a very early stage. It seems like there is a clear discrepancy between the validation and test sets. Could you please confirm if the proposed HERO model shows similar behaviours on these validation and test subsets? Also, would it be possible to get the validation performances of the models? Are the validation scores the same as in the v1 paper?
> 
> You can find the accuracy curves for training / validation / test sets [here](https://drive.google.com/file/d/1YPkalOEoo718sQW0MV1emDAQT6_D_8Kl/view?usp=sharing).
> 
> Thank you for your help in advance.

The predications of val and test is here:
(https://drive.google.com/drive/folders/1x0dLHIRlvQimyRuRSSHMUPTE5HouAOAi?usp=sharing)

**Comment by antoyang** [comment]
It looks from these predictions that your model achieves 74.1 accuracy on the public val which is quite close to its performances on the public test set. I also observed in my experiments significant performance gaps between the val and test sets with various models (more than 20 points of accuracy).
May I know when you plan to release the full val set or the features and code to reproduce the results on this dataset?
